README.md

```diff
@@ -4,19 +4,21 @@
 [![Ruff](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/charliermarsh/ruff/main/assets/badge/v0.json)](https://github.com/charliermarsh/ruff)
 [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/python/black)
 [![pre-commit](https://img.shields.io/badge/pre--commit-enabled-brightgreen?logo=pre-commit&logoColor=white)](https://github.com/pre-commit/pre-commit)
+[![project chat](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://neuroinformatics.zulipchat.com/#narrow/stream/406001-Movement/topic/Welcome!)
 
 # movement
 
 Kinematic analysis of animal 🐝 🦀 🐀 🐒 body movements for neuroscience and ethology research 🔬.
 
 - Read the [documentation](https://neuroinformatics-unit.github.io/movement/) for more information.
 - If you wish to contribute, please read the [contributing guide](./CONTRIBUTING.md).
+- Join our [zulip chat](https://neuroinformatics.zulipchat.com/#narrow/stream/406001-Movement/topic/Welcome!) to interact with the team. We welcome your questions and suggestions.
```

Comment:
```suggestion
- Join our [zulip chat](https://neuroinformatics.zulipchat.com/#narrow/stream/406001-Movement/topic/Welcome!) to interact chat the team. We welcome your questions and suggestions.
```

Happy with chat? Sounds a bit less formal.

Reply:
I prefer "chat" to "interact", but I want to avoid duplicating the word "chat" in the same sentence.
How about "Join us on zulip to chat with the team"?

Reply:
Perfect.